lib/Sema/CSDiagnostics.cpp

```diff
@@ -2236,6 +2237,13 @@ bool MissingMemberFailure::diagnoseAsError() {
   auto tryTypoCorrection = [&] {
     TC.performTypoCorrection(getDC(), DeclRefKind::Ordinary, baseType,
                              defaultMemberLookupOptions, corrections);
+    // If isForKeyPathDynamicMemberLookup we are including the
```

Comment:
(If you're going to comment on the 'if' statement, then might as well actually explain it.)

```suggestion
    // If locator points to the member found via key path dynamic member lookup,
```